src/seed_tools/utils/study_json_utils.ts

```diff
@@ -50,7 +51,7 @@ export function parseStudies(
 ): Study[] {
   const jsonStudies = JSON5.parse(
     studyArrayString,
-    jsonStudyReviever.bind(null, options),
+    jsonStudyReviewer.bind(null, options),
```

Comment:
it should actually be `jsonStudyReviver` 😄 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#reviver

Reply:
O_o. Reverted back

Reply:
no, I had a typo here actually, let's fix it: `jsonStudyReviever` -> `jsonStudyReviver`.

Reply:
Fixed ;)